templates/default/typescript/www/images/.taco-ignore

```diff
@@ -1,3 +0,0 @@
-This is a dummy file with the sole purpose of allowing to commit its parent directory (which would otherwise be empty) to git source control.
-
-It is not intended to be part of the template and will be excluded from the archive process (when doing "gulp prepare-templates").
```

Comment:
We discussed this offline, but putting answer here for others: we still need need the taco-ignore file for the www/scripts/ folder of the TypeScript template